src/common/core/core.js

```diff
@@ -1480,6 +1548,117 @@ define([
+
+        /**
+         * Gathers the paths of the mixin nodes associated with the node
+         * that were defined specifically for the given node.
+         *
+         * @param {module:Core~Node} node - the node in question.
+         *
+         * @return {string[]} The paths of the own mixins in an array.
+         *
+         * @func
+         */
+        this.getOwnMixinPaths = core.getOwnMixinPaths;
+
+        /**
+         * Gathers the mixin nodes associated with the node.
+         *
+         * @param {module:Core~Node} node - the node in question.
+         *
+         * @return {Object<string, module:Core~Node>} The dictionary of the mixin nodes keyed by their paths.
+         *
+         * @func
+         */
+        this.getMixinNodes = core.getMixinNodes;
+
+        /**
+         * Gathers the mixin nodes associated with the node that were defined specifically for the given node.
+         *
+         * @param {module:Core~Node} node - the node in question.
+         *
+         * @return {Object<string, module:Core~Node>} The dictionary of the own mixin nodes keyed by their paths.
+         *
+         * @func
+         */
+        this.getOwnMixinNodes = core.getOwnMixinNodes;
+
+        /**
+         * Removes a mixin from the mixin set of the node.
+         *
+         * @param {module:Core~Node} node - the node in question.
+         * @param {string} mixinPath - the path of the mixin node.
+         *
+         * @func
+         */
+        this.delMixin = core.delMixin;
+
+        /**
+         * Adds a mixin to the mixin set of the node.
+         *
+         * @param {module:Core~Node} node - the node in question.
+         * @param {string} mixinPath - the path of the mixin node.
+         *
+         * @func
+         */
+        this.addMixin = core.addMixin;
+
+        /**
+         * Removes all mixins for a given node.
+         *
+         * @param {module:Core~Node} node - the node in question.
+         *
+         * @func
+         */
+        this.clearMixins = core.clearMixins;
+
+        /**
+         * Searches for the closest META node of the node in question.
+         * @param {module:Core~Node} node - the node in question
+         *
+         * @return {Object<string, module:Core~Node>} Returns the closest Meta node that is a base of the given node
+         * plus it returns all the mixin nodes associated with the base in a path-node dictionary.
+         *
+         * @func
+         */
+        this.getBaseTypes = core.getBaseTypes;
+
+        /**
+         * Checks if a node could be added as a mixin to the given node.
+         *
+         * @param {module:Core~Node} node - the node in question.
+         * @param {string} mixinPath - the path of the mixin node.
+         *
+         * @return {Object} - Returns if the mixin could be added, or the reason why it is not.
+         *
+         * @func
+         */
+        this.canSetAsMixin = core.canSetAsMixin;
```

Comment:
Would change it to `Checks if a node could be added as a mixin to the given path.`